src/components/groupSet/GroupSetSelect.js

```diff
@@ -21,37 +21,54 @@ const GroupSetSelect = ({
     value,
     allowNone,
     onChange,
-    errorText,
     className,
 }) => {
     const { nameProperty } = useCachedDataQuery()
     const { loading, error, data } = useDataQuery(ORG_UNIT_GROUP_SETS_QUERY, {
         variables: { nameProperty },
     })
 
+    const ITEM_NONE = { id: 'none', name: i18n.t('None') }
```

Comment:
Please separate the GroupSetSelect fixes to a separate PR. Is there a jira ticket for it? Also, there's a lint error for the useMemo (missing dependency).

We can still release this at the same time as the popup style fixes using a release branch, but the fixes seem too unrelated to be in the same PR.